lib/ansible/modules/cloud/azure/azure_rm_resourcegroup_info.py

```diff
@@ -16,9 +16,9 @@
 
 DOCUMENTATION = '''
 ---
-module: azure_rm_resourcegroup_facts
+module: azure_rm_resourcegroup_info
 
-version_added: "2.1"
+version_added: "2.9"
```

Comment:
Version added should not change when renaming a module.

Also, the symlink for the original deprecated name is missing.

Reply:
yes, I actually wasn't sure about that, since azure_rm_resourcegroup_info is introduces, and azure_rm_resourcegroup_facts becomes obsolete.